TerraformLibrary/terraformlibrary.py

```diff
@@ -68,12 +69,15 @@ class TerraformLibrary:
     with any terraform script. 
 
     """
-    def _run_command(self, command: str):
+
+    def _run_command(self, command: str, include_stderr: bool = False):
         process = subprocess.run(
-            command, shell=True, capture_output=True
+            command, shell=True, stdout=subprocess.PIPE, stderr=(subprocess.STDOUT if include_stderr else subprocess.PIPE)
```

Comment:
Sorry for the dumb question, does this mean we are treating in a different way `stdout` and `stderror` when `include_stderr` is `true`?

Reply:
The `subprocess.run` function has a fairly weird interface that evolved over time with weird overlapping behavior.

`capture_output=True` is equivalent to `stdout=subprocess.PIPE, stderr=subprocess.PIPE` and results in stdout and stderr being available separately on the result object.

`stderr=STDOUT` instead streams the stderr to the same buffer as stdout and interleaves it as though it were a terminal. 

So the include_stderr option switches between both modes here.